### DISCUSSION TITLE
Running Contiki container on macOS

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#1605] [Q&A]
I'm trying to run Contiki on macOS using the instructions [here](https://github.com/contiki-ng/contiki-ng/wiki/Docker). In there it reads "The Dockerfile can be found in the Contiki-NG repository under tools/docker", which I'm using. When trying to run the container I get the message:

"WARNING: The requested image's platform (linux/386) does not match the detected host platform (linux/amd64) and no specific platform was requested"

**Comment by pjonsson** [comment]
The warning is harmless. Version 4.8 will come with a 64-bit docker image though.
